README.md

```diff
@@ -1 +1,270 @@
+  * Appears to be from [the `flysystem` module's `hook_install()` implementation](https://git.drupalcode.org/project/flysystem/-/blob/cf46f90fa6cda0e794318d04e5e8e6e148818c9a/flysystem.install#L27-32)
+    where it tries to ensure that all schemes defined are in a state ready
+    to be used; however, all the modules are not yet enabled (in the
+    particular case, `islandora` is not actually enabled, so the `fedora` driver is unknown),
+    and so leading to this message being emitted. The `islandora` module
+    _is_ enabled by the time the command exits, so this message should be
+    ignorable.
+* `[warning] Could not find required jsonld.settings to add default RDF namespaces.`
+  * Appears to be from [the `islandora` module's `hook_install()` implementation](https://github.com/Islandora/islandora/blob/725b5592803564c9727e920b780247e45ecbc9a4/islandora.install#L8-L13)
+    where it tries to alter the `jsonld` module's `jsonld.settings` config
+    object to add some namespaces; however, because the configs are not yet
+    installed when installing the modules with `--existing-config`, it fails
+    to find the target configuration to alter it. As exported, the
+    `jsonld.settings` already contains the alterations (at time of writing),
+    so this warning should be ignorable.
+
+In summary: These two messages seem to be ignorable.
+
+### Ongoing Project Maintenance
+
+It is anticipated that [Composer](https://getcomposer.org/) will be used to manage Drupal and its extensions,
+including Islandora's suite of modules. The Drupal project has already documented many of the interactions in this
+space, so we will just list and summarize them here:
+
+[Using Composer to Install Drupal and Manage Dependencies](https://www.drupal.org/docs/develop/using-composer/manage-dependencies)
+* The "install" describing:
+  * Composer's `create-project` command, which we describe above being used to install
+    using this "starter site" project; and,
+  * The `drush site:install`/`drush si` command, described above being used to install Drupal via the command line.
+* "manage[ment]", describing:
+  * Composer's `require` command, used to add additional dependencies to your project
+  * Updating, linking out to additional documentation for Drupal's core and modules:
+    * [Updating Drupal core via Composer](https://www.drupal.org/docs/updating-drupal/updating-drupal-core-via-composer)
+    * [Updating Modules and Themes using Composer](https://www.drupal.org/docs/updating-drupal/updating-modules-and-themes-using-composer)
+
+    Generally, gets into using Composer's `update` command to update extensions according to the specifications in
+    the `composer.json`, and Composer's `require` command to _change_ those specifications when necessary to cross
+    major version boundaries.
+
+It is also recommended to monitor and/or subscribe to Drupal's security advisories to know when it might be
+necessary to update.
+
```

Comment:
Should we also add some info about lock files and conflicts - those are some of the most common issues with composer? Also maybe composer update --lock ?

Reply:
The same argument could be made that the upstream docs that we're pointing at should, so... not sure that we should bother getting into specifics when we don't have any others here in this regard.

Reply:
This section has been removed from this PR, being re-added in #4 ... so follow-ups should occur over there.